### ISSUE TITLE
Unexpected token import at import fs from 'fs-extra'

### ISSUE THREAD

**Issue by chauhanshubham** [gatsbyjs/gatsby#123]
My gatsby version 0.7.7 and throws this error while running the command 
`gatsby new bloggy https://github.com/gatsbyjs/gatsby-starter-blog`

```
import fs from 'fs-extra'
^^^^^^

SyntaxError: Unexpected token import
    at exports.runInThisContext (vm.js:54:16)
    at Module._compile (module.js:375:25)
    at Module._extensions..js (module.js:406:10)
    at Object.require.extensions.(anonymous function) [as .js] (/usr/local/lib/node_modules/gatsby/node_modules/babel-core/lib/api/register/node.js:214:7)
    at Module.load (/usr/local/lib/node_modules/gatsby/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/gatsby/bin/new.coffee:13:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/gatsby/bin/new.coffee:35:4)
    at Module._compile (module.js:399:26)
    at Object.loadFile (/usr/local/lib/node_modules/gatsby/node_modules/coffee-script/lib/coffee-script/register.js:16:19)
    at Module.load (/usr/local/lib/node_modules/gatsby/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/gatsby/bin/gatsby-new.js:5:1)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Function.Module.runMain (module.js:431:10)
    at startup (node.js:141:18)
```

**Comment by KyleAMathews** [comment]
Hi is this from a global install e.g. "npm install -g gatsby"?

**Comment by chauhanshubham** [comment]
so, I tried the installation, using global install and install from source, and can't remember which would have caused that to happen.
Does that makes a difference? Any ideas please let me know. I'll try once again and, keep you posted on this.

**Comment by KyleAMathews** [comment]
Should be fixed in 0.8, let me know if not.
